doc/nrf/libraries/bluetooth/services/fast_pair/adv_manager.rst

```diff
@@ -0,0 +1,253 @@
+.. _bt_fast_pair_adv_manager_readme:
+
+Fast Pair Advertising Manager
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Fast Pair Advertising Manager is a helper module for the :ref:`bt_fast_pair_readme` that provides management of the Fast Pair advertising set.
+The module implements a trigger-based system for controlling Fast Pair advertising state that allows client modules to request advertising with their preferred configuration.
+It also defines the use case layer that provides implementation of specific advertising requirements for supported use cases.
+
+Overview
+********
+
+The Fast Pair Advertising Manager module provides the following key features:
+
+* **Trigger-based advertising control**: Allows multiple components to request the start of advertising using registered triggers.
+* **Trigger-based advertising configuration**: Provides flexible configuration of the advertising parameters such as the pairing mode and RPA rotation behavior based on trigger requirements.
+* **Payload synchronization**: Ensures advertising payload updates are synchronized with RPA rotations.
+* **Extension compatibility**: Is compatible with the Fast Pair extensions that are supported by the :ref:`bt_fast_pair_readme` module (for example, the Find My Device Network extension).
+
+  .. include:: /includes/fast_pair_fmdn_rename.txt
+
+* **Use case support**: Provides tailored behavior for specific Fast Pair device types (for example, locator tags).
+
+.. note::
+   The module is designed to work in cooperative thread context.
+
+.. _bt_fast_pair_adv_manager_config:
+
+Configuration
+*************
+
+The Fast Pair Advertising Manager is enabled with the :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER` Kconfig option.
```

Comment:
```suggestion
To enable the Fast Pair Advertising Manager, use the :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER` Kconfig option.
```

Reply:
Are you sure? We have a similar sentence in the integration guide, and I thought the module documentation should look a bit differently (e.g. more descriptive sentences instead of user-oriented call-to-action instructions).